packages/riipen-ui/src/components/Switch.jsx

```diff
@@ -0,0 +1,219 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import _JSXStyle from "styled-jsx/style";
+
+import ThemeContext from "../styles/ThemeContext";
+
+const Switch = props => {
+  const {
+    checked: checkedProp,
+    classes,
+    color,
+    disabled,
+    id,
+    onChange,
+    required,
+    size,
+    value
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const [checked, setChecked] = useState(checkedProp);
```

Comment:
I don't think we actually want this. You mixed up two forms of input here, both "controlled" and "uncontrolled" inputs.

https://stackoverflow.com/questions/42522515/what-are-react-controlled-components-and-uncontrolled-components

By having a `checked` prop you are enabled a controlled component, but by having this state transition you are also enabling a uncontrolled component. When build react we should never mix the two as it can have really weird side effects.

Looking at our other components, most notable `<Checkbox>`, I see we have opted for controlled components only. We should update this component to be more like the checkbox. Only have the `checked` prop and not this internal state. You will also need to update the demo pages slightly to accommodate for this. Checkout the checkbox demo pages for good examples on how to handle this.